### DISCUSSION TITLE
Property CIV/maybe.1 should be deleted

### DISCUSSION THREAD

**Discussion by rivettp** [edmcouncil/fibo#1413] [General] [answered]
Whether one thing "may be" another is built into OWL unless there is a disjointWith triple

_Originally posted by @rivettp in https://github.com/edmcouncil/fibo/pull/828_

**Comment by trypuz** [comment]
We have 4 "may be" properties in FIBO:
- https://spec.edmcouncil.org/fibo/ontology/BP/SecuritiesIssuance/EquitiesIPOIssuance/mayBe
- https://spec.edmcouncil.org/fibo/ontology/DER/CreditDerivatives/CreditDefaultSwaps/mayBe
- https://spec.edmcouncil.org/fibo/ontology/SEC/Debt/AssetBackedSecurities/PoolBackedSecurities/mayBe
- https://spec.edmcouncil.org/fibo/ontology/CIV/Funds/CIV/mayBe.1

What do we do with them? Shall we remove them all and introduce restrictions with some domain-specific properties everywhere "may be" is currently used (perhaps with min 0 and max n cardinalities)?

**Comment by ElisaKendall** [comment]
There are more than this - if you search for mayBecome, you'll find some additional ones.  We should probably eliminate them, but we need to review these ontologies to see whether some other property name makes sense or they should be deleted altogether.  These are all in provisional ontologies, nothing that has been reviewed by an FCT recently.

**Comment by rivettp** [comment]
See also #843 for mayBecome. 
It seems these are attempts to tie changes in lifecycle state with additional classes that would get added that it must/must not have a certain lifecycle state. So maybe it needs an enhancement to Lifecycles. Or alternatively a restriction on the class. 
It also begs the question as to when individuals are created. If a Mortgage only comes into existence once the contract is executed then does it make sense to have any states prior to that? Or is there a single individual that starts as an instance of Application and changes class to Mortgage once executed?
All this goes well beyond what OWL provides so maybe we should have an orthogonal level of process documentation associated with anything that is subject to a lifecycle.

**Comment by trypuz** [comment, marked as answer]
CIV/maybe.1 has been deleted.
